src/services/expensesService.js

```diff
@@ -0,0 +1,142 @@
+'use strict';
+
+const { v4: uuidv4 } = require('uuid');
+const { DataTypes } = require('sequelize');
+const { sequelize } = require('../dataBase');
+
+const Expenses = sequelize.define('Expenses', {
+  id: {
+    type: DataTypes.STRING,
+    primaryKey: true,
+  },
+  userId: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+  },
+  spentAt: {
+    type: DataTypes.DATE,
+    allowNull: false,
+  },
+  title: {
+    type: DataTypes.TEXT,
+    allowNull: false,
+    defaultValue: DataTypes.NOW,
+  },
+  amount: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+  },
+  category: {
+    type: DataTypes.TEXT,
+    allowNull: false,
+  },
+  note: {
+    type: DataTypes.TEXT,
+    allowNull: false,
+  },
+}, {
+  tableName: 'expenses',
+  updatedAt: false,
+  createdAt: false,
+});
+
+const getAllExpenses = async({ userId, categories, from, to }) => {
+  let allExpenses = await Expenses.findAll();
+
+  if (categories) {
+    allExpenses = allExpenses.filter(
+      expense => categories.includes(expense.category)
+    );
+  }
+
+  if (userId) {
+    allExpenses = allExpenses.filter(
+      expense => expense.userId === +userId
+    );
+  }
+
+  if (from && to) {
+    allExpenses = allExpenses.filter(
+      item => item.spentAt > from && item.spentAt < to);
```

Comment:
Query from the database only expenses that you actually need instead of querying all and then filtering them.
https://sequelize.org/docs/v6/core-concepts/model-querying-basics/